test/support/helpers/permit-licence.helper.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+/**
+ * @module PermitLicenceHelper
+ */
+
+const PermitLicenceModel = require('../../../app/models/permit-licence.model.js')
+const { generateLicenceRef } = require('./licence.helper.js')
+
+/**
+ * Add a new licence into the licence table in the permit schema
+ *
+ * If no `data` is provided, default values will be used. These are
+ *
+ * - `licenceStatusId` - [int]
+ * - `licenceTypeId` - [int]
+ * - `licenceRegimeId` - [int]
+ * - `licenceSearchKey` - [null]
+ * - `isPublicDomain` - [null]
+ * - `licenceStartDt` - [null]
+ * - `licenceEndDt` - [null]
+ * - `licenceRef` - [randomly generated - 01/123]
+ * - `licenceSummary` - [null]
+ * - `metadata` - [null]
+ * - `dateLicenceVersionPurposeConditionsLastCopied` - [null]
+ * - `dateGaugingStationLinksLastCopied` - [null]
+ * - `dateCreated` - new Date()
+ * - `dateUpdated` - new Date()
+ *
+ * @param {Object} [data] Any data you want to use instead of the defaults used here or in the database
+ *
+ * @returns {Promise<module:PermitLicenceModel>} The instance of the newly created record
+ */
+async function add (data = {}) {
+  const insertData = defaults(data)
+
+  return PermitLicenceModel.query()
+    .insert({ ...insertData })
+    .returning('*')
+}
+
+/**
+ * Returns the defaults used
+ *
+ * It will override or append to them any data provided. Mainly used by the `add()` method, we make it available
+ * for use in tests to avoid having to duplicate values.
+ *
+ * @param {Object} [data] Any data you want to use instead of the defaults used here or in the database
+ */
+function defaults (data = {}) {
+  const defaults = {
+    dateLicenceVersionPurposeConditionsLastCopied: null,
+    dateGaugingStationLinksLastCopied: null,
+    endDate: null,
+    licenceDataValue: null,
+    licenceSearchKey: null,
+    licenceStatusId: 1,
+    licenceTypeId: 8,
+    licenceRef: generateLicenceRef(),
+    licenceRegimeId: 1,
+    licenceSummary: null,
+    metadata: null,
+    publicDomain: null,
+    startDate: null
```

Comment:
Again just checking. Normally if a field is nullable we don't bother setting a default. We just have these for fields which don't allow nulls but don't have a default, or where in 100% cases we expect something to be there.

If we strip the defaults that are set to null, do the tests still pass?